detections/endpoint/windows_rdp_file_execution.yml

```
@@ -0,0 +1,96 @@
+data_source: []
+name: Windows RDP File Execution
+id: 0b6b12b9-8ba9-48fe-b3b8-b4e3e1cd22b4
+version: 1
+date: '2024-11-21'
+author: Michael Haag, Splunk
+type: TTP
+status: production
+description: The following analytic detects when a Windows RDP client attempts to execute an RDP file from a temporary directory, downloads directory, or Outlook directories. This detection is significant as it can indicate an attempt for an adversary to delivery a .rdp file, which may be leveraged by attackers to control or exfiltrate data. If confirmed malicious, this activity could lead to unauthorized access, data theft, or further lateral movement within the network. 
+search: '| tstats count min(_time) as firstTime max(_time) as lastTime from datamodel=Endpoint.Processes 
+  where Processes.process IN ("*\\AppData\\Local\\Temp\\*", "*\\AppData\\Local\\Microsoft\\Outlook\\*", "*\\Content.Outlook\\*", "*\\Downloads\\*")
+  AND Processes.process="*.rdp*"
+  by Processes.process Processes.process_name Processes.user Processes.dest Processes.parent_process_name Processes.parent_process
+  | `drop_dm_object_name(Processes)`
+  | `security_content_ctime(firstTime)`
+  | `security_content_ctime(lastTime)`
+  | eval 
+  execution_type=case(
+      match(process, "\\\\Temp\\\\.*\\.(zip|7z|rar|gz|tar|iso|img|vhd|vhdx).*\\.*\\.rdp"), "temp_archive_execution",
```

Adding a couple of other extensions for delivery, namely "tgz" and "cab" 

```suggestion
      match(process, "\\\\Temp\\\\.*\\.(zip|7z|rar|cab|tgz|gz|tar|iso|img|vhd|vhdx).*\\.*\\.rdp"), "temp_archive_execution",
```